pkg/policyengine/PolicyDispatcher.go

```diff
@@ -214,11 +214,29 @@ func connPolicyFromBlob(blob io.Reader) (*policytypes.ConnectivityPolicy, error)
 	return connPolicy, nil
 }
 
-func (pH *PolicyHandler) AddLBPolicy(lbPolicy *LBPolicy) error {
+func lbPolicyFromBlob(blob io.Reader) (*LBPolicy, error) {
+	lbPolicy := &LBPolicy{}
+	err := json.NewDecoder(blob).Decode(lbPolicy)
+	if err != nil {
+		plog.Errorf("failed decoding load-balancing policy: %v", err)
+		return nil, err
+	}
+	return lbPolicy, nil
+}
+
+func (pH *PolicyHandler) AddLBPolicy(policy *api.Policy) error {
```

Comment:
Can you put a comment before those functions.

Reply:
Done